portfolio.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Siko Norbert's Portfolio</title>
+    <link href="portfolio.css" rel="stylesheet">
+</head>
+<body>
+    <header class="cellular">
```

Comment:
- [ ] You have a horizontal scrollbar on your page. Kindly fix the issue.

https://user-images.githubusercontent.com/70483835/185588732-43e8fa08-c7c8-4fa4-8428-37ba5b829b21.mp4